hadoop-plugin/src/main/groovy/com/linkedin/gradle/azkaban/AzkabanPlugin.groovy

```diff
@@ -216,16 +231,32 @@ class AzkabanPlugin implements Plugin<Project> {
    */
   AzkabanProject readAzkabanProject(Project project) {
     def pluginJson = readAzkabanPluginJson(project);
-    if (pluginJson == null) {
-      throw new GradleException("\n\nPlease run \"gradle writeAzkabanPluginJson\" to create a default .azkabanPlugin.json file in your project directory which you can then edit.\n")
+    AzkabanProject azkProject = makeDefaultAzkabanProject(); //changed makeAzkabanProject here to makeDefaultAzkabanProject()
```

Comment:
makeDefaultAzkabanProject() is the overridden method in li-hadoop-plugin. That's the reason I updated it.